src/vdaf.rs

```diff
@@ -164,7 +164,10 @@ where
     /// State of the Aggregator during the Prepare process.
     type PrepareStep: Clone + Debug;
 
-    /// The type of messages exchanged among the Aggregators during the Prepare process.
+    /// The type of message broadcast by each aggregator at each round.
+    type PrepareBroadcast: Clone + Debug + ParameterizedDecode<Self::PrepareStep> + Encode;
```

Comment:
Interestingly, DAP's model of communication would suggest naming these the other way around. In each DAP round, the leader sends to all helpers (i.e. broadcasts) a combined `PrepareMessage`. Each helper will then compute its next preparation state & `PrepareBroadcast` message by calling `prepare_step`, and send to the leader (i.e. unicast) that `PrepareBroadcast` message. The leader will then combine these messages into a `PrepareMessage` to prepare for the next round of communication.

I see the VDAF model though--I suppose the VDAF model is that every participant broadcasts their `PrepareBroadcast` message to all other participants, who then combine all of the broadcasted messages into a `PrepareMessage`.

I don't want to bikeshed too hard on naming, if for no other reason than DAP currently has exactly one helper so the distinction between broadcast & unicast is somewhat arbitrary. If other folks are weirded out by this too, my suggestion would be something like `PrepareMessage` and `CombinedPrepareMessage`? [also, have we considered if DAP's strategy of having only the leader produce combined messages is OK--could it give an evil leader some advantage if they sent a different `PrepareMessage` than would be honestly computed? probably this has already been thought about.]

Reply:
👍🏻 to using `PrepareShare`/`PrepareMessage`, IMO this choice makes things understandable both in the VDAF's "broadcast" communication model & DAP's "star" communication model. (plus, matching the specification will likely make things easier to understand for folks attempting to translate between the two)

Reply:
Done.